### ISSUE TITLE
Report minimum channel size when opening a channel

### ISSUE THREAD

**Issue by bolatovumar** [LN-Zap/zap-desktop#2694]
It would be nice if Zap reported the minimum channel size that a node you are trying to connect to will accept so you don't end-up with this error message:

![Screen Shot 2019-08-06 at 4 59 03 PM](https://user-images.githubusercontent.com/1934678/62546254-d0774680-b86b-11e9-9d03-ff89a26c9c81.png)

## Detailed Description

It would be nice if Zap reported the minimum amount the channel can opened for and set it in the amount field so you don't up with the error message in the screenshot above.

![Screen Shot 2019-08-06 at 5 12 38 PM](https://user-images.githubusercontent.com/1934678/62547224-9149f500-b86d-11e9-8312-532f9a42425e.png)

## Context

This change would prevent users trying to open channels that are too small in size.

## Possible Implementation

I'm not sure if this will work but could we ping the peer we are trying to connect to and find out the minimum channel size before allowing the user to proceed to the final screen?

## Your Environment

- Zap version: 0.5.2-beta and 0.6.0-alpha

**Comment by JimmyMow** [comment]
@bolatovumar as far as I'm aware this is not possible with the current LND API. Maybe the API has changed with 0.7.1, what do you have in mind?

**Comment by bolatovumar** [comment]
@JimmyMow yeah, I figured this wasn't possible with the current API (I couldn't find anything that would suggest otherwise). But it's good to keep in mind as a possible enhancement for the future.

I guess it the meantime instead of showing that error modal we could show a helpful modal with the same error but which would also allow opening a channel with whatever the minimum amount is given the user has enough on-chain funds. It would probably also be useful to allow the user to adjust the amount inline in the same modal if the user wants to open a channel for a larger amount.

This way you can still open a channel in one click but for a different amount instead of going through the process of creating a new channel all over again.

**Comment by JimmyMow** [comment]
Nice idea @bolatovumar! We have a global modal we use for more information and error. We can easily show this to the user with an action button that takes them to the channel form with the minimum and peer information set!
